### PR TITLE
MET-4964: Sandbox - Add home page

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: projects/sandbox/src/app/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div class="home">
+  <div class="home-header">
+    <h1>Welcome to<br />Metis Sandbox!</h1>
+    <div class="circle-container-outer">
+      <ul class="circle-container">
+        <li class="orb-container"><a routerLink="/new" class="nav-orb labelled upload-orb"></a></li>
+        <li class="orb-container">
+          <a routerLink="/dataset" class="nav-orb labelled progress-orb"></a>
+        </li>
```
Comment (JoanaCMS) [inline projects/sandbox/src/app/home/home.component.html:9]: I like that you can directly create or track a dataset from the home menu. However, I personally don't know what each (green) symbol means. Is it possible to show a message when we hover on top of each symbol?
"Create a dataset" "Track a dataset" etc..

File: projects/sandbox/src/app/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      </ul>
+    </div>
+  </div>
+  <div class="home-content">
+    <span>
+      The Sandbox allows you to:
+    </span>
+    <ol class="list">
+      <li>
+        test to see if your data will work within the Metis flow
```
Comment (JoanaCMS) [inline projects/sandbox/src/app/home/home.component.html:22]: Personal taste, but could you start each sentence with a capital letter?
"Test to see..."
"Get insight into..."
"Preview your records..."

File: projects/sandbox/src/app/home/home.component.html (hunk 1 above)
Comment (andyjmaclean) [reply projects/sandbox/src/app/home/home.component.html:15]: Done!

File: projects/sandbox/src/app/home/home.component.html (hunk 2 above)
Comment (andyjmaclean) [reply projects/sandbox/src/app/home/home.component.html:28]: I like proper capitalisation too, but technically those lines don't start sentences, so shouldn't be capitalised!

**Review by JoanaCMS — APPROVED** [review]

**Pull request merged**